const checkPermission = async (req, res, next) => {
  try {
    
  } catch (error) {
    console.log(error);    
  }
};
